health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PopInbox.js

```
@@ -405,17 +410,18 @@ const PopInbox = () => {
       // reorder the status count to show pending for validation on top
+      const statusOrderMap = Object.fromEntries(
+        config.filterConfig.filterOptions.map(option => [option.status, option.order])
+      );
+  
+      // Reorder the status count based on `order` from filterConfig
       const reorderedStatusCount = Object.fromEntries(
         Object.entries(data?.StatusCount || {}).sort(([keyA], [keyB]) => {
-          if (keyA === "PENDING_FOR_VALIDATION") return -1;
-          if (keyB === "PENDING_FOR_VALIDATION") return 1;
-          return 0;
+          return (statusOrderMap[keyA] || Infinity) - (statusOrderMap[keyB] || Infinity);
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)